src/flags/sorting.rs

```diff
@@ -330,67 +300,70 @@ mod test_sort_column {
     }
 
     #[test]
-    fn test_from_config_empty() {
-        let yaml_string = "---";
-        let yaml = YamlLoader::load_from_str(yaml_string).unwrap()[0].clone();
-        assert_eq!(None, SortColumn::from_config(&Config::with_yaml(yaml)));
-    }
+    fn test_from_config_none_column() {
```

Comment:
Change the test name back to `test_from_config_empty`. It is more consistent and essentially testing the same thing as the previous implementation.